packages/application-components/src/components/dialogs/confirmation-dialog/confirmation-dialog.spec.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { SecondaryButton } from '@commercetools-frontend/ui-kit';
 import { render, wait, fireEvent } from '../../../../test-utils';
 import ConfirmationDialog from './confirmation-dialog';
 
-const DialogController = props => {
+type DialogControllerProps = {
+  children: (renderProps: {
+    isOpen: boolean;
+    setIsOpen: React.Dispatch<boolean>;
```

Comment:
I can't find anything in google (not redux related) about React.Dispatch. can you enlighten?

Reply:
It's in the `@types/react` package:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bb36642c70f366af454e1bf67471fbfdf0b9bdc8/types/react/index.d.ts#L810

Reply:
ah okay. weird

would it be more readable to just put (value: A) => void ?

Reply:
This is just for a test btw, I just went with the same API used by `useState`.

Also, if you have TS editor installed, you can see the type signatures:
<img width="529" alt="image" src="https://user-images.githubusercontent.com/1110551/62467966-380e9280-b795-11e9-9343-522d478e43c4.png">